gnovm/cmd/gno/util.go

```diff
@@ -343,3 +343,19 @@ func prettySize(nb int64) string {
 	}
 	return fmt.Sprintf("%.1f%c", float64(nb)/float64(div), "kMGTPE"[exp])
 }
+
+// isExecutionSuccessfulWithinTimeout checks if the execution of a given exec.Cmd
+// is successful within a specified timeout.
+func isExecutionSuccessfulWithinTimeout(cmd *exec.Cmd, timeout time.Duration) bool {
+	errc := make(chan error, 1)
+	go func() {
```

Comment:
This routine is never closed if the timeout triggers

Reply:
Removed